.github/ISSUE_TEMPLATE/00_BB_creation_template.yml

```diff
@@ -363,4 +363,29 @@ body:
     label: Url to Documentation for the Release Process Badge
     placeholder: 
   validations:
-    required: false
\ No newline at end of file
+    required: false
+
+- type: textarea
+  id: sys_context
+  attributes:
+    label: System Context
+    description: OS and runtime/framework requirements eg. AGL, QNX, ROS-based, container runtime, web assembly, web service
+    placeholder: 
+  validations:
+    required: false
+
+- type: dropdown
+  id: state
+  attributes:
+    label: State
+    description: Please select the current state of the BB
+    multiple: true
+    options:
+    - Incubating (no code yet)
+    - Implementation started
+    - First public release available
+    - Used in production by 1 OEM
+    - Used in production by >1 OEM
+    - Abandoned
```

Comment:
Maybe we need a kind of dictionary here? The options in the issue were a suggestion.

Example:

    - type: dropdown
      id: state
      attributes:
        label: State
        description: Please select the current state of the BB
        multiple: true
        options:
            incubating: "Incubating (no code yet)"
            alpha: "Implementation started"
            stable: "Release, tested, usage unknown."
            in_series: "Used in production by at least 1 OEM (relevant for on-board SW)"
            multi_use: "Used in production by at more than 1 OEM (relevant for on-board SW)"
            abandoned: "Project is not maintained anymore."

Reply:
We also need a date of last change of the state

    - type: datetime
      id: state_since
      attributes:
        label: Last change of state
        description: When was the state changed.
      validations:
        required: false